code/Model/RedirectorPage.php

```diff
@@ -39,19 +41,27 @@ class RedirectorPage extends Page
 
     private static $has_one = [
         "LinkTo" => SiteTree::class,
+        "LinkToFile" => File::class
+    ];
+
+    private static $owns = [
+        'LinkToFile'
```

Comment:
I don't think a redirect should own the file. If it's not published separately then the redirect won't work, but that'd be expected from my perspective.

Reply:
@michalkleiner Thanks, removed the `$owns`